include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -0,0 +1,1897 @@
+     *
+     * Linear in the size of lit.
+     *
+     * ### Exceptions
+     *
+     * Throws std::invalid_argument if any character is not '0' or '1'.
+     */
+    template <size_t N>
+    constexpr void assign(char const (&lit)[N])
+    {
+        static_assert(N <= bit_capacity + 1, "Length of string literal exceeds capacity of dynamic_bitset.");
+        assert(lit[N - 1] == '\0');
+        uint64_t value{};
+
+        for (size_t i = 0; i != N - 1; ++i)
+        {
+            if (lit[i] == '0')
+            {
+                value <<= 1;
+            }
+            else if (lit[i] == '1')
+            {
+                value <<= 1;
+                value |= 1u;
+            }
+            else
+            {
+                throw std::invalid_argument{"The string to construct a dynamic_bitset from may only contain 0 and 1."};
+            }
+        }
+
+        *this = value;
+        resize(N - 1);
+    }
+
+    /*!\brief Assign from `std::initializer_list`.
+     * \param[in] ilist A `std::initializer_list` of value_type.
+     *
+     * \details
+     *
+     * ### Complexity
+     *
+     * Linear in the size of `ilist`.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr void assign(std::initializer_list<value_type> const ilist) noexcept
+    {
+        assign(std::ranges::begin(ilist), std::ranges::end(ilist));
+    }
+
+    /*!\brief Assign with `count` times `value`.
+     * \param[in] count Number of elements.
+     * \param[in] value The initial value to be assigned.
+     *
+     * \details
+     *
+     * ### Complexity
+     *
+     * Linear in `count`.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    constexpr void assign(size_type const count, value_type const value) noexcept
+    {
+        clear();
+        auto tmp = view::repeat_n(value, count);
+        assign(std::ranges::begin(tmp), std::ranges::end(tmp));
+    }
+
+    /*!\brief Assign from a different range.
+     * \tparam other_range_t The type of range to be inserted; must satisfy std::ranges::InputRange and `value_type`
+     *                       must be constructible from reference_t<other_range_t>.
+     * \param[in]      range The sequences to construct/assign from.
+     *
+     * \details
+     *
+     * ### Complexity
+     *
+     * Linear in the size of `range`.
+     *
+     * ### Exceptions
+     *
+     * No-throw guarantee.
+     */
+    template <std::ranges::InputRange other_range_t>
+    //!\cond
+        requires std::Constructible<value_type, reference_t<other_range_t>>
+    //!\endcond
+    constexpr void assign(other_range_t && range) noexcept
+    {
+        assign(std::ranges::begin(range), std::ranges::end(range));
+    }
+
+    /*!\brief Assign from pair of iterators.
+     * \tparam begin_it_type Must satisfy std::ForwardIterator and the `value_type` must be constructible from
+     *                       the reference type of begin_it_type.
```

Comment:
can you please check all comments again, i.e. that code expressions are wrapped in a `code`-block. So that it is consistent through the documentation